digger.lua

```diff
@@ -186,25 +89,23 @@ local function fullDig()
   for zCount = 1, zMagnitude do
     for yCount = 1, yMagnitude do
 
-      moveX(xSign * xMagnitude, true)
+      movement.moveX(xSign * xMagnitude, {dig, checkAndRefuel})
 
       if yCount < yMagnitude then
         xSign = xSign * -1
-        moveY(ySign, true)
+        movement.moveY(ySign, {dig, checkAndRefuel})
       end
     end
 
-    if zCount < zMagnitude then 
+    if zCount < zMagnitude then
       ySign = ySign * -1
       xSign = xSign * -1
-      moveZ(zSign, true)
+      movement.moveZ(zSign, dig, {checkAndRefuel})
```

Comment:
looks like a typo, dig should be in callbacks

Reply:
nah check out `moveZ`